src/gc/gcinterface.h

```diff
@@ -463,6 +465,8 @@ class IGCHandleManager {
     virtual Object* GetDependentHandleSecondary(OBJECTHANDLE handle) = 0;
 
     virtual Object* InterlockedCompareExchangeObjectInHandle(OBJECTHANDLE handle, Object* object, Object* comparandObject) = 0;
+
+    virtual bool HandleAsyncPinnedHandles() = 0;
```

Comment:
I agree - it's cryptic enough to me that I don't really know what it does or what it should be named, though...